--- conf/ss5.init	2009-07-11 10:29:22.000000000 +0200
+++ conf/ss5.init.dlucio	2010-05-05 23:17:05.000000000 +0200
@@ -1,9 +1,26 @@
 #!/bin/sh
+# ss5		This shell script takes care of starting and stopping
+#		SS5 SOCKS proxy
 #
 # chkconfig: 345 20 80
 # description: This script takes care of starting \
 #              and stopping ss5
 #
+# pidfile: /var/run/ss5.pid
+# config: /etc/ss5/ss5.conf
+
+
+### BEGIN INIT INFO
+# Provides: ss5
+# Required-Start: $network
+# Required-Stop: $network
+# Should-Start: $named 
+# Should-Stop: $named 
+# Short-Description: Starts the ss5 daemon
+# Description: SS5 is a full-featured SOCKS4/5
+#              
+### END INIT INFO
+
 
 OS=`uname -s`
 if [ $OS = "Linux" ] || [ $OS = "SunOS" ]; then
